src/main/java/org/logstash/tcp/InputLoop.java

```diff
@@ -222,9 +226,8 @@ public void exceptionCaught(final ChannelHandlerContext ctx, final Throwable cau
             private boolean silentException(final Throwable ex) {
                 if (ex instanceof IOException) {
                     final String message = ex.getMessage();
-                    if ("Connection reset by peer".equals(message)) {
-                        return true;
-                    }
+                    if (message != null)
+                        return silentErrs.stream().anyMatch(message::contains);
```

Comment:
It seems to me we can simplify the `Array` since both elements have `Connection reset` message:
```suggestion
                    if (Objects.nonNull(message))
                        return message.contains(CONNECTION_RESET_ERROR_MSG);
```

Reply:
if there is `StringUtils` available, using `nonBlank` or `isEmpty` would be also nice.
```
if (ex instanceof IOException == FALSE) {
  return false;
}

final String errorMessage = ex.getMessage();
return Objects.nonNull(errorMessage) ? errorMessage.contains(CONNECTION_RESET_ERROR_MSG): false;
```

Reply:
updated